WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewModel.swift

```diff
@@ -62,9 +82,62 @@ final class InPersonPaymentsMenuViewModel {
             self?.showWebView = nil
         })
     }
+
+    /// This method is just a helper for debugging, we may use it for populating different Banner content based on the fetched objects count
+    ///
+    func displayIPPFeedbackBannerIfEligible() {
+        if isCODEnabled {
+            let resultsCount = resultsController.fetchedObjects.count
+            let numberOfTransactions = 10
+
+            print("IPP transactions within 30 days: \(resultsCount)")
+            print(resultsController.fetchedObjects.map {
+                ("OrderID: \($0.orderID) - PaymentMethodID: \($0.paymentMethodID) - DatePaid: \(String(describing: $0.datePaid))")
+            })
+
+            if resultsCount == 0 {
+                // TODO: This option should use a different results controller or predicate, as we're looking for 0 orders historically, not within 30 days.
```

Comment:
I'm wondering what would be the best approach here, as the 1st condition is "Stores with COD enabled, but no transactions" we're looking for 0 transactions historically, not within 30 days. 

Should we use two different results controller, or dynamically modify the predicate somehow? I'd think that if we fetch once to see if there's 0 or any orders, we could use that same results controller to filter down the latest 30 days without the need to fetch again.